content/en/tracing/manual_instrumentation/dotnet.md

```diff
@@ -9,36 +11,67 @@ further_reading:
-      tag: 'Documentation'
-      text: 'Implement Opentracing across your applications'
     - link: 'tracing/visualization/'
       tag: 'Documentation'
       text: 'Explore your services, resources, and traces'
 ---
+This page details common use cases for adding and customizing observability with Datadog APM.
 
-If you are not using libraries supported by automatic instrumentation (see [Integrations][1]), you can instrument your code manually.
+Add the `Datadog.Trace` [NuGet package][1] to your application. In your code, access the global tracer through the `Datadog.Trace.Tracer.Instance` property to create new spans.
 
-The following example uses the global `Tracer` and creates a custom [span][2] to [trace][3] a web request:
+This is supported on **.NET Framework 4.5** and above on Windows and on **.NET Core 2.1, 3.0, and 3.1** on Windows and Linux.
 
-```csharp
-using Datadog.Trace;
 
-using(var scope = Tracer.Instance.StartActive("web.request"))
+## Adding Tags & Spans
+
+Add custom [span tags][2] to your [spans][3] to customize your observability within Datadog.  The span tags are applied to your incoming traces, allowing you to correlate observed behavior with code-level information such as merchant tier, checkout amount, or user ID.
+
+
+### Add custom span tags
+
+Add custom tags to your spans corresponding to any dynamic value within your application code such as `customer.id`.
+
+Add tags directly to a `Datadog.Trace.Span` object by calling `Span.SetTag()`. For example:
+
+```csharp
+public class ShoppingCartController : Controller
 {
-    var span = scope.Span;
-    span.Type = SpanTypes.Web;
-    span.ResourceName = request.Url;
-    span.SetTag(Tags.HttpMethod, request.Method);
+    private IShoppingCartRepository _shoppingCartRepository;
+
+    [HttpGet]
+    public IActionResult Index(int customerId)
+    {
+        // Access the active scope through the global tracer (can return null)
+        var scope = Tracer.Instance.ActiveScope;
+
+        if (scope != null)
+        {
+            // Add a tag to the span for use in the datadog web UI
+            scope.Span.SetTag("customer.id", customerId.ToString());
+        }
+
+        var cart = _shoppingCartRepository.Get(customerId);
 
-    // do some work...
+        return View(cart);
+    }
 }
 ```
 
+**Note**: `Datadog.Trace.Tracer.Instance.ActiveScope` returns `null` if there is no active span.
 
-### Setting Errors
 
-To recognize and mark errors that occur in your code, utilize the `Span.SetException(Exception)` method available to spans. The method marks the span as an error and adds [related span metadata][4] to provide insight into the exception.
+
+### Adding tags globally to all spans
+
+The `DD_TRACE_GLOBAL_TAGS` environment variable allows setting tags across all generated spans for an application. This can be useful for grouping stats for your applications, datacenters, or any other tags you would like to see within the Datadog UI.
```

Comment:
The dotnet tracer supports the new standard `DD_TAGS`.
```suggestion
The `DD_TAGS` environment variable allows setting tags across all generated spans for an application. This can be useful for grouping stats for your applications, datacenters, or any other tags you would like to see within the Datadog UI.
```